app/scaffold/scaffold.js

```diff
@@ -181,14 +196,19 @@ class Scaffold {
             if (!promise) {
                 return;
             }
+
+            DOM.alertBox("Decrypting");
```

Comment:
Add ellipses here too to stay consistent.

Reply:
Done